relay-server/src/services/store.rs

```diff
@@ -494,6 +494,16 @@ impl StoreService {
             .send_message(topic, organization_id, &message)?;
 
         match &message {
+            KafkaMessage::Metric {
+                message: metric, ..
+            } => {
+                metric!(
+                    counter(RelayCounters::ProcessingMessageProduced) += 1,
+                    event_type = message.variant(),
+                    metric_type = metric.value.variant(),
+                    metric_encoding = metric.value.encoding().unwrap_or(""),
```

Comment:
Could you document these here? https://github.com/getsentry/relay/blob/3dff617a1a5ff01f6617de82807bc50c7d695a2c/relay-server/src/statsd.rs#L557-L562

Also curious that we documented a `namespace` tag there which we apparently dropped.